src/NzbDrone.Core/Download/History/DownloadHistory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using NzbDrone.Core.Datastore;
+
+namespace NzbDrone.Core.Download.History
+{
+    public class DownloadHistory : ModelBase
+    {
+        public DownloadHistoryEventType EventType { get; set; }
+        public int SeriesId { get; set; }
+        public string DownloadId { get; set; }
+        public string SourceTitle { get; set; }
+        public DateTime Date { get; set; }
+    }
```

Comment:
I feel like we should include more details while were at it:
- Protocol
- IndexerId (maybe Indexer name too?)
- DownloadClientId (maybe Download Client name too?)
- Data json with additional details, but in particular:
  - ReleaseInfo (kinda like PendingReleases), it could be useful in calculating upgrade word scores
  - Original Preferred Word score

Reply:
What would we use this for? Unlike History this is meant as an internal tracking of grabs/imports and I'm not opposed to adding more information if we have a goal for it, but right now this is a moving target and delays fixing of the known issues.

Reply:
The existing Grab/Import have the Protocol, indexer & Download client details. I believe it would be prudent to duplicate that here so we have all information about a download here.

ReleaseInfo isn't present anywhere yet, and the usecase is to be able to use all info in the ReleaseInfo/TorrentInfo to calculate upgrade scores that don't rely on title alone, such as the Indexer filter in the profile or other tags (freeleech).   
It's not something we currently have (except the indexer filter), but I worked on it a bit and the full ReleaseInfo was something I missed. It's only applicable for the Grab event, obviously.